base/loading.jl

```diff
@@ -821,10 +821,13 @@ function _require_from_serialized(path::String)
     return _include_from_serialized(path, depmods)
 end
 
+const LOAD_TIMING = Ref{Bool}(false)
```

Comment:
How does this work with regard to thread-safety #41602?

Reply:
Good point. Should be fixed now